NetExericise/NetExericise/Array/Exercise02.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetExericise.Array
+{
+    public class Exercise02
+    {
+        public int MaxTripletSum(int[] arr)
+        {
+            var sortedArr = arr.OrderByDescending(num => num).ToArray();
```

Comment:
Đã dùng Linq thì sao không dùng cho nốt
```suggestion
            var sum = arr.OrderByDescending(num => num).Take(3).Sum();
```

Reply:
Done